request_validator.js

```diff
@@ -28,4 +28,37 @@ const validate = (data) =>
     }
     return errors;
 }
-exports.validate= validate;
\ No newline at end of file
+
+const validateSupportRecord = (data) =>
+{
+    var errors = [];
+    if(Object.keys(data).length < 26)
```

Comment:
With the other form, I know there are conditional routes - does that mean there might be a scenario where front end sends less fields as the "shorter" route was taken or does the front end always send all fields?

Reply:
The front end should send all fields, even if they pass null or blank values for those not required.  We could do some more robust validations here, I know, it was my plan for today before plans changed.